requirements.txt

```diff
@@ -8,23 +7,25 @@ coveralls==1.11.1
 dj-database-url==0.5.0
 Django==3.0.3
 django-heroku==0.3.1
+django-timezone-field==4.0
 djangorestframework==3.11.0
+djangorestframework-simplejwt==4.4.0
 docopt==0.6.2
 entrypoints==0.3
 flake8==3.7.9
 gunicorn==20.0.4
+httpie==2.1.0
 idna==2.8
 mccabe==0.6.1
 Pillow==7.0.0
-promise==2.3
 psycopg2==2.8.4
 pycodestyle==2.5.0
 pyflakes==2.1.1
+Pygments==2.6.1
+PyJWT==1.7.1
```

Comment:
same here, is this needed?

Reply:
PyJWT is the package needed for JWT. I uninstalled it and I could not run tests locally again